openedx.yaml

```diff
@@ -0,0 +1,6 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+oeps: {}
+owner: edx/teaching-and-learning
+obsolete: True
```

Comment:
@e0d: I think Gemnasium reports on this repo.

1. Do you want a ticket to respect this "obsolete" status in Gemnasium, either manually or automatically?
2. Should we require that the README also gets updated with a warning for obsolete repos that their security, or lack there of, is not being tracked and it is likely unsafe to use?  Or, do we just want to clearly document this along with our definition of what it means to be an "obsolete" repo?

FYI: @cpennington